pandas/tests/frame/test_constructors.py

```diff
@@ -306,18 +306,24 @@ def test_constructor_dtype_nocast_view_2d_array(
             assert df2._mgr.arrays[0].flags.c_contiguous
 
     @td.skip_array_manager_invalid_test
-    def test_1d_object_array_does_not_copy(self):
+    def test_1d_object_array_does_not_copy(self, using_copy_on_write):
         # https://github.com/pandas-dev/pandas/issues/39272
         arr = np.array(["a", "b"], dtype="object")
```

Comment:
Or add `copy=False` here to keep the test as is? (and to keep coverage of that case)

Reply:
sure, done